Runs from site root and can execute plain classes with Execute method

lars-erik opened the pull request:
Not sure whether you'd like to keep maintaining this, but with these changes, it'll be easy to create small tasks you want to run without adding web UI or other stuff.
Also a bit "nicer" to run from site root imho. :)

I'll probably be polishing a bit more and/or just doing things differently when using this, so just have a look for now.

lars-erik committed:
Runs from site root and can execute plain classes with Execute(ApplicationContext) method.

sitereactor commented:
Cool ... I haven't looked at this repo in a long time, but I know people still find it and use the example, so this PR is most definitely welcome.

Thanks for sending this :star2: :smile:

lars-erik committed:
Commandline argument for class to execute

lars-erik committed:
Tool can reside at %path% and load the rest from current folder .\bin.

lars-erik committed:
Attempting powershell module with umbraco in appdomain :O

lars-erik committed:
Works on some sites. Crashes on invalid references apparently?

lars-erik committed:
Works on many sites. Must "eager load" some assemblies (pass parameters). Crash in some reflection / typeloader cases. IE. Method not implemented even if it is.

lars-erik committed:
Bloody works! :O

lars-erik committed:
Some docs on the CLI tool.
Loads of failed attemts at getting HttpRuntime/HostingEnvironment up with the right paths and appdomain data

lars-erik committed:
Removed customer-specific build-event
